src/content/1.7/basics/installation/_index.md

```diff
@@ -23,16 +24,65 @@ Here is a selection of free AMP installers:
 * [MAMP](https://www.mamp.info/) (Mac OS X)
 * [Laragon](https://laragon.org/) (Windows)
 
+To install LAMP on your computer follow these steps (tested on Debian Buster).
+- Update your system
+  - `apt update`
+- Install MySQL
+  - `apt install default-mysql-server default-mysql-client`
+- Install Apache server
+  - `apt install apache2`
+- Install PHP
+  - `apt install libapache2-mod-php7.3 php7.3 php7.3-common php7.3-curl php7.3-gd php7.3-imagick php7.3-mbstring php7.3-mysql php7.0-json php7.3-xsl php7.3-intl php7.3-zip`
```

Comment:
While I see the usefulness, I'm not sure if its fitting to provide a detailed guide on how to install the AMP stack on Debian...

Reply:
I followed what Wordpress, Joomla, and Drupal do on their doc :/